backend/tracim_backend/lib/utils/translation.py

```diff
@@ -1,14 +1,19 @@
 # -*- coding: utf-8 -*-
+from enum import Enum
+import glob
 import json
-import os
+from os.path import basename
+from os.path import dirname
+import pathlib
```

Comment:
If you're using `pathlib`, `basename` and `dirname` aren't necessary, the equivalent exist in `pathlib`: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
done.